tests/10apidoc/01register.pl

```diff
@@ -240,23 +239,30 @@ sub matrix_v1_register_user_via_secret
    defined $uid or
       croak "Require UID for matrix_register_user_via_secret";
 
-   my $mac = hmac_sha1_hex(
-      join( "\0", $uid, $password, $is_admin ? "admin" : "notadmin" ),
+   $http->do_request_json(
+      method => "GET",
+      uri    => "/r0/admin/register",
+   )->then(sub{
```

Comment:
sorry, the whitespace conventions for sytest are... unusual.

this should have a space inside the parens:

`)->then( sub {`